widget/tree.go

```diff
@@ -586,6 +586,7 @@ type treeNode struct {
 	depth   int
 	hovered bool
 	icon    fyne.CanvasObject
+	isLeaf  bool
```

Comment:
Let's keep consistent with the rest of Tree and use `isBranch`

Reply:
yeah, as above, good catch